website/static/js/profile.js

```diff
@@ -264,6 +264,13 @@ BaseViewModel.prototype.handleSuccess = function() {
 BaseViewModel.prototype.handleError = function(response) {
     var defaultMsg = 'Could not update settings';
     var msg = response.message_long || defaultMsg;
+
+    for(var i = 0 ;i<this.contents().length; i++){
+        if(this.contents()[i].institution() === ''){
```

Comment:
This doesn't feel right; the `BaseViewModel` shouldn't have to know about its subclasses' fields (e.g. `institution`). Can you use knockout-validation on the institution field instead? Or expose a hook to subclasses that allows them to perform additional validation?